webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/LengthUnit.scala

```diff
@@ -0,0 +1,81 @@
+package com.scalableminds.webknossos.datastore.models
+
+import com.scalableminds.util.enumeration.ExtendedEnumeration
+import play.api.libs.json.{Format, JsError, JsPath, JsResult, JsString, JsSuccess, JsValue, JsonValidationError}
+
+object LengthUnit extends ExtendedEnumeration {
+  type LengthUnit = Value
+  val ym, zm, am, fm, pm, nm, µm, mm, cm, dm, m, hm, km, Mm, Gm, Tm, Pm, Em, Zm, Ym, Å, in, ft, yd, mi, pc = Value
+
+  override def fromString(s: String): Option[Value] =
+    s match {
+      case "ym" | "yoctometer" => Some(ym)
+      case "zm" | "zeptometer" => Some(zm)
+      case "am" | "attometer"  => Some(am)
+      case "fm" | "femtometer" => Some(fm)
+      case "pm" | "picometer"  => Some(pm)
+      case "nm" | "nanometer"  => Some(nm)
+      case "µm" | "micrometer" => Some(µm)
+      case "mm" | "millimeter" => Some(mm)
+      case "cm" | "centimeter" => Some(cm)
+      case "dm" | "decimeter"  => Some(dm)
+      case "m" | "meter"       => Some(m)
+      case "hm" | "hectometer" => Some(hm)
+      case "km" | "kilometer"  => Some(km)
+      case "Mm" | "megameter"  => Some(Mm)
+      case "Gm" | "gigameter"  => Some(Gm)
+      case "Tm" | "terameter"  => Some(Tm)
+      case "Pm" | "petameter"  => Some(Pm)
+      case "Em" | "exameter"   => Some(Em)
+      case "Zm" | "zettameter" => Some(Zm)
+      case "Ym" | "yottameter" => Some(Ym)
+      case "Å" | "angstrom"    => Some(Å)
+      case "in" | "inch"       => Some(in)
+      case "ft" | "foot"       => Some(ft)
+      case "yd" | "yard"       => Some(yd)
+      case "mi" | "mile"       => Some(mi)
+      case "pc" | "parsec"     => Some(pc)
+      case _                   => None
```

Comment:
Shouldn't it be `Some(None)` here, or is it implicitly converted to an Option because thats what the method expects as a return type and scala does implicit conversion magic?